app/models/user.rb

```diff
@@ -5,6 +5,12 @@ class User < ApplicationRecord
 
   validates :name, presence: true
 
+  def is_e2e_user?
+    ENV.fetch("E2E_EMAILS", "")
```

Comment:
would `E2E_USER_EMAILS` be a littler clearer?

Reply:
Good plan - I've updated the Helm charts too https://github.com/alphagov/govuk-helm-charts/pull/3592